backend/routes/userRouter.js

```diff
@@ -9,102 +9,47 @@ const user = require('../data/helpers/userModel');
 /* ---------- Endpoints for /api/user ---------- */
 
 /* GET (list) */
-router.get('/', (req, res) => {
+router.get( '/', (req, res) => {
 
   user.get()
-    .then((users) => {
+    .then( (users) => {
       res.json(users);
     })
-    .catch((err) => {
+    .catch( (err) => {
       res.status(500).json({ error: "User information could not be retrieved." });
     });
 });
 
 
 /* GET by id */
-router.get('/:id', (req, res) => {
+router.get( '/:id', (req, res) => {
   const { id } = req.params;
 
   user.get(id)
-    .then((users) => {
-      if (users) {
+    .then( (users) => {
```

Comment:
Since this is going to be one user, it would make more sense just to call these variables `user` instead of `users`